sdc/tests/tests_perf/test_perf_utils.py

```diff
@@ -197,7 +197,9 @@ def get_times(f, *args, iter_number=5):
 
 class TestResults:
     perf_results_xlsx = 'perf_results.xlsx'
+    perf_results_html = 'perf_results.html'
```

Comment:
I vote for waiting for https://github.com/IntelPython/sdc/pull/347 to apply these changes over this PR.

Reply:
@densmirn , @PokhodenkoSA ,
Then I think result's dump to html should be implemented in https://github.com/IntelPython/sdc/pull/347
And in current PR we can revert changes with html and merge it.

Reply:
Please resolve merge conflicts.